config/locales/ar.yml

```diff
@@ -55,8 +55,12 @@ ar:
       description: صفحة من كتاب %{title} من تأليف %{author}
       empty_page: الصفحة فارغة
       error_in_loading_page: حدث خطأ أثناء تحميل الصفحة
+      hide_tashkeel: إخفاء التشكيل
+      hide_tashkeel_tooltip: إخفاء علامات التشكيل
```

Comment:
These translation are not used, could you please remove them. Also, the 2 below these ones.